huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,45 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md
+# refer to the above page to pull latest Tensorflow image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training:2.6.3-gpu-py38-cu112-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# version args
+ARG TRANSFORMERS_VERSION
+ARG DATASETS_VERSION
+ARG PYTHON=python3
+
+# uninstall ipython jupyter utils due to incompatibility with typing-extensions==3.7.4.3
+# https://github.com/huggingface/huggingface_hub/blob/main/setup.py:19
+RUN pip uninstall -y IPython sparkmagic jupyter-console ipywidgets ipykernel hdijupyterutils jupyter sagemaker-studio-analytics-extension qtconsole notebook autovizwidget widgetsnbextension
+
+# install Hugging Face libraries and its dependencies
+RUN pip install --no-cache-dir \
+    transformers[sklearn,sentencepiece]==${TRANSFORMERS_VERSION} \
+    datasets==${DATASETS_VERSION} \
+    tensorflow-addons==0.12.0 \
+    psutil \
+    # TODO: Remove upgrade statements once packages are updated in base image
+    "Pillow>=9.0.0" \
+    "typing-extensions==3.7.4.3" \
+    "bokeh==2.3.3"
+
```

Comment:
I imagine most of these are fixed in TF 2.6.3 since it was just released, can we try removing some of these temporary packages?

Reply:
removed Pillow 9.0.1 since it's already installed in the base image
the other two are for HF specifically, modified the comment for it.